superset/config.py

```diff
@@ -204,7 +204,10 @@ def _try_json_readsha(filepath: str, length: int) -> str | None:
 # `SQLALCHEMY_ENGINE_OPTIONS = {"isolation_level": "READ COMMITTED"}`
 # Also note that we recommend READ COMMITTED for regular operation.
 # Find out more here https://flask-sqlalchemy.palletsprojects.com/en/3.1.x/config/
-SQLALCHEMY_ENGINE_OPTIONS = {}
+# for example, if you don't want to use a connection pool, uncomment those next 2 lines
+# from sqlalchemy.pool import NullPool
+# SQLALCHEMY_ENGINE_OPTIONS = {"poolclass": NullPool}
+
```

Comment:
```suggestion
SQLALCHEMY_ENGINE_OPTIONS = {}
```
I'm guessing that you still want the default value.